simulation/modules/geometry.py

```diff
@@ -143,3 +149,16012 @@ def check_abort(obj, event):
         iren.Initialize()
         ren_win.Render()
         iren.Start()
+
+
+def man_geo():
```

Comment:
This function really inflates the additions number for this pr. Is that okay or should we modify this?